lib/Data/Validate/Sanctions/Fetcher.pm

```diff
@@ -0,0 +1,121 @@
+package Data::Validate::Sanctions::Fetcher;
+
+use strict;
+use warnings;
+
+use LWP::UserAgent;
+use Text::CSV;
+use XML::Simple qw(:strict);
```

Comment:
Can provide your candidate, if you don't like this?
My experience with XML is very low.